src/frequenz/sdk/timeseries/battery_pool/_metric_calculator.py

```diff
@@ -229,7 +229,7 @@ def calculate(
             total_capacity += usable_capacity
 
         return (
-            None
+            Sample(timestamp, None)
             if timestamp == _MIN_TIMESTAMP
```

Comment:
I think in all these places, we have to use `datetime.now(tz=timezone.utc)`, because otherwise, we'll send out the `_MIN_TIMESTAMP`, which is year 1 I think.